<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewServerWidget</class>
 <widget class="QWidget" name="NewServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>184</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelServerAddress">
     <property name="text">
      <string>Server Address</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="serverAddressLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelUsername">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KLineEdit" name="userNameLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KLineEdit" name="passwordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="KPushButton" name="verifyButton">
     <property name="text">
      <string>Check Connection</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="verifyLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
